README-WAF.md

```diff
@@ -62,7 +62,18 @@ waf list
 
 ## Program groups ##
 
-The programs in ardupilot are categorized into the following groups:
+Program groups are used to represent a class of programs. They can be used to
+build all programs of a certain class without having to specify each program. A
+program can belong to more than one group.
```

Comment:
Do we have such a case? Where a program belongs to more than one group? If yes, in what folder does it end up?